src/utils/index.js

```diff
@@ -90,14 +90,14 @@ class Utils {
     return google.plus({ version: 'v1', auth });
   }
 
-  /*
+  /**
   * RandomPassword
   * @description generates a random password
   * @returns { String } random password
   */
-  static randomPassword() {
+  /* static randomPassword() {
```

Comment:
Nice job overall ugo. I understand that the email feature is not ready instead of doing this, you can just add the generated password as a part of the response, instead of having to change much of the code to accommodate for that, this can still work tho, I think anna would give the final review.

Reply:
My tests pass locally, so I'll approve this